test/JDBC/input/BABEL-4390-vu-prepare.sql

```diff
@@ -56,4 +56,19 @@ CREATE PROCEDURE babel_4390_prepare_p9 AS
 BEGIN
     EXEC('DROP PROCEDURE sp_testlinkedserver');
 END
+GO
+
+CREATE SCHEMA babel_4390_test_schema;
+GO
+
+CREATE PROCEDURE babel_4390_test_schema.xp_qv AS
+BEGIN
+    SELECT 1;
+END
+GO
+
+CREATE PROCEDURE babel_4390_test_schema.sp_addlinkedsrvlogin AS
+BEGIN
+    SELECT 1;
+END
```

Comment:
can we add positive test where we show that we are able to dro such procedures in user defined schema